### PR TITLE
Fixed URL regex

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [x] In Progress - Waiting for some clarifications
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30770

## Description
Improved the regex for URL indicator to support fragment support without no trailing slash.

## Screenshots
URL extraction while the fixed one is 'URL2':
image
image

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/ReleaseNotes/2_7_3.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Types
+- **urlRep** - Improved the regex for URL indicator to support fragment support without no trailing slash.
```
Comment (yaakovi) [inline Packs/CommonTypes/ReleaseNotes/2_7_3.md:3]:
```suggestion
- **urlRep** - Improved the regex for URL indicator to support fragment support without trailing slash.
```

**Review by yaakovi — APPROVED** [review]
looks ok, lets run the performance tests before merging

**Comment by reutshal** [comment]
Playbook addition:
image

File: Packs/CommonTypes/ReleaseNotes/2_7_3.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Types
+- **urlRep** - The URL regex now supports fragment without trailing slash.
```
Comment (bakatzir) [inline Packs/CommonTypes/ReleaseNotes/2_7_3.md:3]: Customers know it as URL
```suggestion
- ** URL** - The URL regex now supports fragment without trailing slash.
```

**Review by bakatzir — APPROVED** [review]

**Pull request merged**